packages/video_player/video_player/lib/video_player.dart

```diff
@@ -639,7 +646,10 @@ class _VideoPlayerState extends State<VideoPlayer> {
   Widget build(BuildContext context) {
     return _textureId == VideoPlayerController.kUninitializedTextureId
         ? Container()
-        : _videoPlayerPlatform.buildView(_textureId);
+        : Transform.rotate(
+            angle: widget.controller.value.rotation,
+            child: _videoPlayerPlatform.buildView(_textureId),
+          );
```

Comment:
This is the heart of this PR (the rest is plumbing so Android plugin can set rotation if rotationDegrees == 180). Transforming the widget doesn't seem terrible, but I'd prefer to fix it at the Java level.